README.md

```diff
@@ -65,21 +65,37 @@ Load Marduk platform specific OpenWrt configuration for Pistachio.
         Target Profile (Basic platform profile for Marduk)  ---> 
             (X) Basic platform profile for Marduk with TI cc2520
             ( ) Basic platform profile for Marduk with Cascoda ca8210
+            ( ) Wifi testing profile for Marduk with TI cc2520
 
-You can also load Marduk platform specific OpenWrt configuration for Pistachio by adding following into .config file.
+You can also load Marduk platform (with TI cc2520) specific OpenWrt configuration for Pistachio by adding following into .config file.
```

Comment:
Update this line "Alternatively, you can also load Marduk platform with TI cc2520 specific OpenWrt configuration for IMG Pistachio by adding following into .config file:"